app/src/hooks/connectedWeb3.tsx

```diff
@@ -31,12 +31,15 @@ export const useConnectedWeb3Context = () => {
 
   return context
 }
-
+interface Props {
+  children?: React.ReactNode
+  setStatus?: any
+}
 /**
  * Component used to render components that depend on Web3 being available. These components can then
  * `useConnectedWeb3Context` safely to get web3 stuff without having to null check it.
  */
-export const ConnectedWeb3: React.FC = props => {
+export const ConnectedWeb3: React.FC = (props: Props) => {
```

Comment:
to fix that prop error you were getting you can do `React.FC<Props>`

Reply:
ohh man you are saving me again :D Thanks man you'r awesome!